<ui version="4.0" >
 <class>ReservationsView</class>
 <widget class="QWidget" name="ReservationsView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptionsView</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" />
   </item>
   <item>
    <widget class="TitleWidget" native="1" name="titleWidget" />
   </item>
   <item>
    <widget class="QStackedWidget" name="controlsStack" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageIntro" >
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Une reservation assure qu'un client DHCP se voit assigner tjours la même adresse IP.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblVariant" >
         <property name="text" >
          <string>Pour ajouter une réservation, cliquez sur Nouvelle réservation.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Remarque : une exclusion empêche un client DHCP d'obtenir une adresse d'une plage d'adresses spécifiques. Les plages d'exclusion peuvent être définies dans un pool d'adresses.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Pour plus d'informations sur les réservations, consultez la documentation de votre DHCPd.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddReservation" >
         <property name="text" >
          <string>Ajouter réservation</string>
         </property>
         <property name="icon" >
          <iconset resource="dyprosium.qrc" >:/images/Resources/computer_add.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageView" >
      <layout class="QVBoxLayout" >
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QTableWidget" name="tableReservations" >
           <property name="editTriggers" >
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors" >
            <bool>true</bool>
           </property>
           <property name="selectionMode" >
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <column>
            <property name="text" >
             <string>Nom</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>IP Address</string>
            </property>
           </column>
           <column>
            <property name="text" >
             <string>MAC Address</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QToolButton" name="btnAdd" >
             <property name="text" >
              <string/>
             </property>
             <property name="icon" >
              <iconset resource="dyprosium.qrc" >:/images/Resources/add.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnRemove" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="icon" >
              <iconset resource="dyprosium.qrc" >:/images/Resources/delete.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnEdit" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="icon" >
              <iconset resource="dyprosium.qrc" >:/images/Resources/pencil.png</iconset>
             </property>
             <property name="checked" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="dyprosium.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>btnAddReservation</sender>
   <signal>clicked()</signal>
   <receiver>btnAdd</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel" >
     <x>393</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
